### PR TITLE
Closes #300: Fix lint errors

### LINKED ISSUE #300: Fix lint errors
### Vision statement / What / Requirements
CI is failing on several lint errors.
### Impact
These failures block merging.
### Acceptance criteria
Fix the errors so that CI is passing again.

### PR DESCRIPTION
**Pull request by dnarcese**
### Pull Request checklist

- [x] This PR includes thorough **tests** or an explanation of why it does not
ci lint errors
- [x] This PR includes a **CHANGELOG entry** or does not need one
- [x] The **UI tests** are passing after this PR (`./gradlew connectedAmazonWebViewDebugAndroidTest`)
- [x] I have considered adding **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-denied`)

### CODE REVIEW

File: app/lint.xml [hunk 1]
```diff
@@ -4,6 +4,9 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <lint>
 
+    <!-- This doesn't affect us, as we are not distributing through the Play Store -->
+    <issue id="ExpiredTargetSdkVersion" severity="informational"/>
```
Comment (mcomella) [inline app/lint.xml]: nit: this should be below the "DISABLED CHECKS" comment.

File: app/build.gradle [hunk 2]
```diff
@@ -122,7 +122,6 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation "androidx.browser:browser:$androidx_version"
```
Comment (mcomella) [inline app/build.gradle]: Nice find! This also seems to be unused: do you mind removing it and double checking nothing looks weird at runtime?

File: app/build.gradle [hunk 3]
```diff
@@ -122,7 +122,6 @@ dependencies {
...
     implementation "com.google.android.material:material:$androidx_version"
-    implementation "androidx.leanback:leanback:$androidx_version"
```
Comment (mcomella) [inline app/build.gradle]: nit: we need to update our licenses for our changed dependencies.

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -9,6 +9,9 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" /> <!-- Used by sentry. -->
 
+    <!--To fix lint error 'Hardware feature touchscreen not explicitly marked as optional'-->
+    <uses-feature android:name="android.hardware.touchscreen" android:required="true" />
```
Comment (mcomella) [inline app/src/main/AndroidManifest.xml]: I think you don't need this commit: I checked out the commit above this, ran lint, and didn't get the error anymore. I think removing the leanback dependency fixed this failure.

File: app/build.gradle (hunk 2 above)
Comment (dnarcese) [reply app/build.gradle]: It used in `SafeIntentKtTest`

**Pull request merged**